### PR TITLE
feat(files): open log file when extension is executed

### PR DESCRIPTION
**Pull request by safeamiiir**
- Select the file from the file list and open it in the content viewer

### CODE REVIEW

**Review by str-anger — APPROVED** [review]

File: aqueductcore/frontend/src/components/organisms/ExtensionModal/index.tsx [hunk 1]
```diff
@@ -112,8 +115,12 @@ function ExtensionModal({ isOpen, handleClose, selectedExtension }: ExtensionMod
         )
     }, [selectedExtension, selectedAction])
 
-    function handleOnCompletedExtensionExecution(executeExtension: EXECUTE_EXTENSION_TYPE) {
+    async function handleOnCompletedExtensionExecution(executeExtension: EXECUTE_EXTENSION_TYPE) {
         handleClose()
```
Comment (jatinriverlane) [inline aqueductcore/frontend/src/components/organisms/ExtensionModal/index.tsx:119]: Currently, on successful execution of extension, the extension window gets closed, but the data in the form stays in the modal. i.e. after execution, when we open the extension modal again, the previous data is still present. 

Logic for clearing the fields can be added here.

**Review by jatinriverlane — APPROVED** [review]
Thanks @safeamiiir

File: aqueductcore/frontend/src/components/organisms/ExtensionModal/index.tsx (hunk 1 above)
Comment (safeamiiir) [reply aqueductcore/frontend/src/components/organisms/ExtensionModal/index.tsx:119]: You're right.
That's actually part of the enhancement ticket, that we can move it to a new one to be fair! could you please do that?
I just didn't want to touch on that in this PR.

Comment (safeamiiir) [reply aqueductcore/frontend/src/components/organisms/ExtensionModal/index.tsx:119]: will be handled here #162

**Pull request merged**